src/components/VContentSwitcher/VSearchTypes.vue

```diff
@@ -84,7 +86,7 @@ export default defineComponent({
      * For example having all content types under `content.types` and making them filterable,
      * like `const additional = [...content.types.filter(i => i.status === ADDITIONAL)]`.
      */
-    if (isDev) {
+    if (featureFlagStore.featureState('models_3d') === ON) {
```

Comment:
In #1285, I moved the enabling of 3d models to the constants module (by adding or not adding it to the list of the media types). Do you think we should maybe get that merged first?
I wonder if we should have both video and 3d models switchable? Or should they have different status?

Reply:
> In https://github.com/WordPress/openverse-frontend/pull/1285, I moved the enabling of 3d models to the constants module (by adding or not adding it to the list of the media types). Do you think we should maybe get that merged first?

FWIW this makes more sense to me! Push the flag check as high up in the app as possible and the rest can just operate as usual, just iterating through the list of supported media types as it does today.

>  I wonder if we should have both video and 3d models switchable? Or should they have different status?

Does video require more than a single PR? What is the current goal for it, I wasn't even aware that it was something we were working on :sweat_smile: If it's multiple PRs of work then adding a flag would make sense (it doesn't cost anything to put it behind a flag, other than the amount of time spent cleaning the flag up after the feature is stable).

Reply:
We can break them up into two feature flags with different statuses.